test/90_1_prm/test_appendix_g_prm.rb

```diff
@@ -330,6 +333,80 @@ def check_lpd(prototypes_base)
     end
   end
 
+  # Check lighting occ sensor
+  #
+  # @param prototypes_base [Hash] Baseline prototypes
+  def check_light_occ_sensor(prototypes,prototypes_base)
+    light_sch = {}
+    prototypes.each do |prototype, model_proto|
+      building_type, template, climate_zone, mod = prototype
+      # Define name of spaces used for verification
```

Comment:
That's a detail, but this comment should probably be on line 344.

Reply:
Modified.